src/main/java/net/sf/jabref/importer/fileformat/BibTeXMLImporter.java

```diff
@@ -37,9 +37,9 @@
  * check here for details on the format
  * http://www.ecst.csuchico.edu/~jacobsd/bib/formats/endnote.html
  */
-public class BibteXMLImporter extends ImportFormat {
+public class BibTeXMLImporter extends ImportFormat {
```

Comment:
The authors also name it that strange :see_no_evil: - see http://bibtexml.sourceforge.net/ and https://github.com/Zearin/BibTeXML